installer/roles/ansible-service-broker-setup/files/provision-ansible-service-broker.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
This looks good, most of this was from https://github.com/fusor/catasb , but this looks like an improvement/update which may be able to be used in full instead of being modified.